src/js/defaultTheme/styles.scss

```diff
@@ -45,6 +45,10 @@
 
 }
 
+.mstr-hide-controls {
+    animation: hide-bars-animation 0.4s forwards;
```

Comment:
While this works, it's an animation and it will push forward until it's done or it can stop instantly if the user mouses away while the animation is midflight. I suggest using `transition` instead of `animation` and change values between the active & inactive state like so:

```css
.player .bottombar {
    opacity: 1;
    transform: none;
    transition: 0.4s ease-in-out;
}

.player.is-hidden .bottombar {
    opacity: 0;
    transform: translateY(100%);
}
```